$OpenBSD: patch-lib_packet-io_c,v 1.3 2014/10/07 09:55:10 kirby Exp $

evutil.h already included in packet-io.h
Merged upstream 620a6aaf3095dc467f88a4f049d526eeca39cd61

--- lib/packet-io.c.orig	Fri Dec 13 13:37:55 2013
+++ lib/packet-io.c	Fri Dec 13 13:38:15 2013
@@ -21,7 +21,6 @@
 
 
 #include <glib.h>
-#include <evutil.h>
 
 #include "packet.h"
 #include "packet-io.h"
